spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -453,6 +454,7 @@
 "http://www.cecill.info/licences/Licence_CeCILL-C_V1-en.txt": CECILL-1.0
 "http://www.cecill.info/licences/Licence_CeCILL_V2.1-en.txt": CECILL-2.1
 "http://www.gnu.org/copyleft/lesser.html": LGPL-3.0-only
+"http://www.gwtproject.org/terms.html": Apache-2.0 AND BSD-3-Clause AND CC0-1.0 AND EPL-1.0 AND LGPL-2.1-only AND MPL-1.1
```

Comment:
Hm, shall we accept the risk that the terms change?

Reply:
Those terms have not changed since they were introduced in 2013 see the [Wayback Machine](https://web.archive.org/web/20130611094055/http://www.gwtproject.org/terms.html)

Reply:
I think @fviernau is right here: It's OK to map links to upstream / original license texts to SPDX ids, but it's not OK to do the same for project-specific links to license texts, as these might change unnoticed.

Reply:
yeah, my point was also about the future. What do you think?

Reply:
@sschuberth  Just so we are aligned you not OK with http://www.gwtproject.org/terms.html but you would be OK if the link was http://www.gwtproject.org/gwt-licenses-1.0.html?

Reply:
Basically yes. I would feel even safer if http://www.gwtproject.org/gwt-project-license-1.0.html was http://www.gwtproject.org/apache-2.0-license.html, obviously, but I'd also be fine with http://www.gwtproject.org/gwt-project-license-1.0.html.

*Edit:* Aaaah, you've changed http://www.gwtproject.org/gwt-project-license-1.0.html into http://www.gwtproject.org/license.html while I wrote this. No, I would *not* be fine with http://www.gwtproject.org/license.html as it says nothing in the URL about which version of the project it applies to.

Reply:
@sschuberth  You are changing what was previously accepted as a mapping - following you above argumentation we would need to remove the following lines:

- [L440: http://go.microsoft.com/fwlink/?LinkId=329770](https://github.com/oss-review-toolkit/ort/blob/master/spdx-utils/src/main/resources/declared-license-mapping.yml#L440)
-[L441: http://polymer.github.io/LICENSE.txt](https://github.com/oss-review-toolkit/ort/blob/master/spdx-utils/src/main/resources/declared-license-mapping.yml#L441)
- [L442: http://svnkit.com/license.html](https://github.com/oss-review-toolkit/ort/blob/master/spdx-utils/src/main/resources/declared-license-mapping.yml#L442)
- [L447: http://www.gnu.org/copyleft/lesser.html](https://github.com/oss-review-toolkit/ort/blob/master/spdx-utils/src/main/resources/declared-license-mapping.yml#L447) as lesser has changed in the past from LGPL-2.0 to LGPL-2.1 to LGPL-3.0 
- [L474: https://raw.github.com/RDFLib/rdflib/master/LICENSE](https://github.com/oss-review-toolkit/ort/blob/master/spdx-utils/src/main/resources/declared-license-mapping.yml#L474)

Reply:
I acknowledge that it was "wrong" before, but I believe the discussion we're having is a good one, and we should looking into fixing this / agree how to go forward before we make things worse.

Reply:
Propose we keep the current way of working until https://github.com/oss-review-toolkit/ort/issues/2898 is implemented and then move any "wrong" link mapping into the examples for package curation mapping. This would provide users a clear way to migrate and not break things too much.